assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -110,7 +135,15 @@ const renderView = (attributes, toAttribute) => {
         <div className='EmptyMessage'>
           {__('Block content is empty. Check the block\'s settings or remove it.', 'planet4-blocks-backend')}
         </div> :
-        <Covers {...coversProps} />
+        <div className='covers-container'>
+          <Covers {...coversProps} />
+          {covers.length > 4 && isCarouselLayout && (
+            <CoversCarouselButton direction='prev' handler={() => {}} />
```

Comment:
As I mentioned before, you missed to add this Component. However, I would like to know how are you handle this `handler` method.